doc: update stack quick start README with RBAC Setup

vadasambar opened the pull request:
details
---------
- setting up RBAC helps in accessing resource outside the stack

vadasambar committed:
doc: update stack quick start README with RBAC Setup

details
---------
- setting up RBAC helps in accessing resource outside the stack

Signed-off-by: Suraj Banakar <suraj@infracloud.io>

Commit:
doc: fix typo 'your' -> 'our'

Signed-off-by: Suraj Banakar <suraj@infracloud.io>

displague commented:
Thanks for the documentation addition, @vadasambar.  

I think noting that dependencies should be added to app.yaml after generated and before publishing the stack is a valuable addition.

I think the section you are providing is necessary to include in this document, although we may want to move sections around (later) to make it clearer that these steps should be taken sooner.

suskin commented on README.md:
```diff
@@ -364,5 +364,22 @@ See the `config/stack/overrides` directory for details about where the
 
+### Setup RBAC
+
+We can setup extra permissions to grant access to the resources which are not part of our stack. This can be done by specifing the permissions in the CustomResourceDefinition YAML of our custom resource.
```
Would you be able to break the long lines earlier? (For example, see lines 363-365 above this.) It makes it a little easier to discuss the changes in GitHub, because comments are for a whole line, and the lines are smaller when there are more line breaks. Also, it'll keep the style in the document consistent : )

vadasambar replied:
Makes sense :+1: I will add more line breaks.

vadasambar replied:
Fixed

suskin commented on README.md:
```diff
@@ -364,5 +364,22 @@ See the `config/stack/overrides` directory for details about where the
 
+### Setup RBAC
+
+We can setup extra permissions to grant access to the resources which are not part of our stack. This can be done by specifing the permissions in the CustomResourceDefinition YAML of our custom resource.
```
Nitpick/typo: `specifing` should be `specifying`

vadasambar replied:
That's a typo alright. Thanks for pointing it out :D

vadasambar replied:
Fixed

suskin commented on README.md:
```diff
@@ -364,5 +364,22 @@ See the `config/stack/overrides` directory for details about where the
 
+### Setup RBAC
+
+We can setup extra permissions to grant access to the resources which are not part of our stack. This can be done by specifing the permissions in the CustomResourceDefinition YAML of our custom resource.
```
> This can be done by specifing the permissions in the CustomResourceDefinition YAML of our custom resource.

This seems like it is not entirely accurate. It sounds like this is talking about specifying permissions in the CRD yaml, but that isn't how it works for a stack. We specify the permissions in the stack definition (the content shown below in the example is a stack definition, which is separate from a CRD). One way to word this would be something like:

```
This can be done by specifying the permissions in the stack definition YAML.
```

vadasambar replied:
You are right. I mistook it for a CRD. Thanks for pointing it out!

vadasambar replied:
Fixed

suskin reviewed (changes requested):
Awesome - thanks @vadasambar for working on the improvements! :squid:

I've left a few minor comments. I'm marking this as changes requested for the change to the wording about which YAML is being updated. Also, would you be able to set the base branch to be `master` instead of the release branch? We've been putting all new changes into `master` so far, and then we can just update the release branch after the change is in `master`. The branch management process is not very formal yet : )

Commit:
doc: add line breaks for `Setup RBAC`

details
----------
- makes it easier to review parts on github
- fix typo `specifing` -> `specifying`
- fix `CustomResourceDefinition` -> `stack definition`

Signed-off-by: Suraj Banakar <suraj@infracloud.io>

suskin reviewed (approved):
Looks good to me. Thanks @vadasambar !
